src/components/WalletModal/index.tsx

```diff
@@ -185,9 +128,9 @@ export default function WalletModal({ openSettings }: { openSettings: () => void
         <ThemedText.SubHeader fontWeight={500}>Connect a wallet</ThemedText.SubHeader>
         <IconButton Icon={Settings} onClick={openSettings} data-testid="wallet-settings" />
       </AutoRow>
-      {pendingError ? (
-        pendingConnection && (
-          <ConnectionErrorView openOptions={openOptions} retryActivation={() => tryActivation(pendingConnection)} />
+      {pending.error ? (
+        pending.error && (
```

Comment:
Is this expected? Shouldn't there be just one `pending.error` check?

Reply:
Ahhh thanks for catching, I was refactoring and accidentally replaced checks for connection with checks for error